tools/src/main/scala/caliban/tools/SchemaWriter.scala

```diff
@@ -3,15 +3,17 @@ package caliban.tools
 import caliban.parsing.adt.Definition.TypeSystemDefinition.TypeDefinition._
 import caliban.parsing.adt.Type.{ ListType, NamedType }
 import caliban.parsing.adt.{ Document, Type }
+import caliban.tools.implicits.Implicits._
+import caliban.tools.implicits.ScalarMappings
 
 object SchemaWriter {
 
   def write(
     schema: Document,
-    objectName: String = "",
```

Comment:
Nice that we don't have to mirror Client and Schema anymore.

Reply:
If you mean this writer lambda propagation, like it was before,  I agree)